frontend/src/components/TableList/TableList.css.ts

```diff
@@ -3,7 +3,7 @@ import { recipe } from '@vanilla-extract/recipes'
 export const sessionAttributeRow = recipe({
 	base: {
 		display: 'grid',
-		gridTemplateColumns: `82px 1fr`,
+		gridTemplateColumns: `90px 1fr`,
```

Comment:
Added a little more space to avoid breaking `operationName` awkwardly

<img width="294" alt="Screenshot 2023-07-18 at 2 31 24 PM" src="https://github.com/highlight/highlight/assets/17744174/0fbf509b-4cb1-4c8a-8d2d-7cc14ea57b17">